system/service.py

```diff
@@ -1005,9 +1005,15 @@ def service_enable(self):
                 if self.module.check_mode:
                     self.module.exit_json(changed=True, msg="changing service enablement")
 
-                rc, stdout, stderr = self.execute_command("%s %s=\"%s\"" % (self.sysrc_cmd, self.rcconf_key, self.rcconf_value ) )
+                rc, change_stdout, change_stderr = self.execute_command("%s %s=\"%s\"" % (self.sysrc_cmd, self.rcconf_key, self.rcconf_value ) )
                 if rc != 0:
-                    self.module.fail_json(msg="unable to set rcvar using sysrc", stdout=stdout, stderr=stderr)
+                    self.module.fail_json(msg="unable to set rcvar using sysrc", stdout=change_stdout, stderr=change_stderr)
+
+                # sysrc does not exit with code 1 on permission error => validate successful change using service(8)
+                rc, check_stdout, check_stderr = self.execute_command("%s %s %s" % (self.svc_cmd, self.name, "enabled"))
+                is_enabled = True if rc == 0 else False
```

Comment:
This condition needs to be rewritten, as it doesn't work on python 2.4.

As a result, the tests on Travis fail:

```
$ python2.4 -m compileall -fq -x 'cloud/' .
Compiling ./system/service.py ...
  File "./system/service.py", line 1014
    is_enabled = True if rc == 0 else False
                       ^
SyntaxError: invalid syntax
```